driver-mapping/src/main/java/com/datastax/driver/mapping/Mapper.java

```diff
@@ -357,15 +489,201 @@ public T get(Object... primaryKey) {
-     * the number of columns composing the PRIMARY KEY of the mapped class, or if
-     * at least one of those values is {@code null}.
+     *                                  the number of columns composing the PRIMARY KEY of the mapped class, or if
+     *                                  at least one of those values is {@code null}.
      */
     public ListenableFuture<T> getAsync(Object... primaryKey) {
         return Futures.transform(session().executeAsync(getQuery(primaryKey)), mapOneFunction);
     }
+
+    /**
+     * Set the default save {@link Option} for this object mapper, that will be used
+     * in all save operations. Refer to {@link Mapper#save)} with Option argument
+     * to check available save options.
+     *
+     * @param options the options to set. To reset, use {@link Mapper#resetDefaultSaveOptions}
+     *                instead of putting null argument here.
+     */
+    public void setDefaultSaveOptions(Option... options) {
+        this.defaultSaveOptions = Arrays.copyOf(options, options.length);
+    }
+
+    /**
+     * Reset the default save options for this object mapper.
+     */
+    public void resetDefaultSaveOptions() {
+        this.defaultSaveOptions = null;
+    }
+
+    /**
+     * Set the default delete {@link Option} for this object mapper, that will be used
+     * in all delete operations. Refer to {@link Mapper#delete)} with Option argument
+     * to check available delete options.
+     *
+     * @param options the options to set. To reset, use {@link Mapper#resetDefaultDeleteOptions}
+     *                instead of putting null argument here.
+     */
+    public void setDefaultDeleteOptions(Option... options) {
+        this.defaultDeleteOptions = Arrays.copyOf(options, options.length);
+    }
+
+    /**
+     * Reset the default delete options for this object mapper.
+     */
+    public void resetDefaultDeleteOptions() {
+        this.defaultDeleteOptions = null;
+    }
+
+    /**
+     * An object to allow defining specific options during a
+     * {@link Mapper#save(Object)} or {@link Mapper#delete(Object...)} operation.
+     * <p>
+     * The options will be added as : 'INSERT [...] USING option-name option-value [AND option-name option value... ].
+     */
+    public static abstract class Option {
+
+        static class Ttl extends Option {
+
+            private int ttlValue;
+
+            Ttl(int value) {
+                this.ttlValue = value;
+            }
+
+            /**
+             * Get the TTL value configured in the object.
+             *
+             * @return the TTL value.
+             */
+            public int getValue() {
+                return this.ttlValue;
+            }
+
+            void appendTo(Insert.Options usings) {
+                usings.and(QueryBuilder.ttl(QueryBuilder.bindMarker()));
+            }
+
+            void appendTo(Delete.Options usings) {
+                usings.and(QueryBuilder.ttl(QueryBuilder.bindMarker()));
```

Comment:
Err, in fact I was wrong, the mapper uses positional parameters so it is compatible with 1.2 (what 1.2 does not support are named parameters, which is irrelevant here).
So you initial comment is valid, we should document it and check it somewhere.